### ISSUE TITLE
Suggestion: make it use the phone to send text messages.

### ISSUE THREAD

**Issue by TeMPOraL** [PebbleSeizureDetect/PebbleSeizureDetect#1]
Hey,

Absolutely amazing work, thank you!

I wonder though, why the choice of Twilio for sending text messages instead of using the phone itself? It seems that a phone will more often have mobile service but no Internet access than the other way around. Having the option to use phone's native texting capabilities would make it both easier to deploy and more reliable.

Please let know if you intend on continuing the development of this project, and/or if you plan on accepting pull requests.

Best,
Jacek "TeMPOraL" Złydach.

**Comment by bygRyan** [comment]
Hi Jacek!  Thanks for the feedback.

I would certainly have just used the phone's text messaging system if it were possible.  But unfortunately Apple doesn't allow apps to send text messages (unless you jailbreak your phone).  That is why Twilio is needed.

I think it is possible to send text messages from apps on Android devices, but I do not have an Android device.

I hope this helps to clarify!

Regarding the project: I may make a few updates here and there, yes.  If you would like to contribute, I would be happy to have help!

**Comment by TeMPOraL** [comment]
Hi,

Thanks for the quick response, and explaining the reason for this choice. I'm an Android use myself, hence the confusion.

I'll see if I can put something together and if so, I'll send you a pull request.

Again, thanks for open-sourcing this great watchapp! :).

**Comment by alex182** [comment]
How about using googlevoice to send text messages?  I used it in a few scripts in the past but  I haven't looked into using that functionality of googlevoice for a few years so it may not be viable anymore.

**Comment by bygRyan** [comment]
If either of you guys are able to get it working and can submit a patch, I'd be happy to include it.  I do not have an Android device, so I cannot do it myself.

Thanks for the help!

**Comment by alex182** [comment]
I'll look into it.  Thanks for making this!

**Comment by etjossem** [comment]
Might be able to use an iMessage hook on iOS devices:

https://dl.dropboxusercontent.com/u/2857188/SMSDemo.zip

**Comment by PebbleSeizureDetect** [comment]
Just wanted to reply to say that "SlopeOak" was indeed me -- I was logged in with the wrong account.

If it is indeed possible to use iMessage that would be great.  Though we need Pebble's app to do that for us, otherwise we need to write our own custom iOS app to communicate with Pebble, which would be a pain.

**Comment by shylesh107** [comment]
Sending messages without user intervention is impossible in iOS devices ( unless you are jailbroken ). Android is a different story though.

**Comment by TeMPOraL** [comment]
I spent some time checking out the docs, there is no way I could find that would allow one to send a text message from a pure watchapp. The only option would be to have a companion app that would do the text messaging part.

**Comment by PebbleSeizureDetect** [comment]
Right.  And that was going to require a fair amount of effort on my part.  Twilio costs me like $1/month, so I went with that :)
